idaes_examples/notebooks/docs/unit_models/reactors/plug_flow_reactor_src.ipynb

```diff
@@ -324,20 +324,33 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now, let us add expressions to compute the reactor cooling cost (\\\\$/s) assuming a cost of 2.12E-5 \\\\$/kW, and the heating utility cost (\\\\$/s) assuming 2.2E-4 \\\\$/kW. Note that the heat duty is in units of watt (J/s). The total operating cost will be the sum of the two, expressed in \\\\$/year assuming 8000 operating hours per year (~10\\% downtime, which is fairly common for small scale chemical plants):"
+    "Now, let us add expressions to compute the reactor cooling cost (\\\\$/s) assuming a cost of 2.12E-5 \\\\$/kW, and the heating utility cost (\\\\$/s) assuming 2.2E-4 \\\\$/kW. To calculate cooling cost, it is important to note that the heat duty is not constant through the reactor's length. This is why we utilize the trapezoid rule to calculate the total heat duty of the reactor:$Q=\\Delta x\\big(\\sum_{k=1}^{N-1}(Q_k)+\\frac{Q_N+Q_0}{2}\\big)$ \n",
```

Comment:
I think we should expand this explanation a little to note that not only is the heat duty not necessarily constant but that it is expressed in terms of heat per unit length (unlike a 0D model where it is just heat)